libsolidity/codegen/ir/Common.cpp

```diff
@@ -122,3 +127,20 @@ string IRNames::zeroValue(Type const& _type, string const& _variableName)
 {
 	return "zero_" + _type.identifier() + _variableName;
 }
+
+string sourceLocationComment(langutil::SourceLocation const& _location, IRGenerationContext const& _context)
```

Comment:
Do you really need this overload?

Reply:
It's used by the `IRGeneratorForStatements` in the `emitCode()` (or now `appendCode()` function as this one only has the location (`m_currentLocation`) available and not the AST object it was called upon.

Reply:
ok!